src/entity/author.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Book } from '@/entity/book.entity'
+import { Column, Entity, JoinColumn, OneToMany, PrimaryGeneratedColumn } from 'typeorm'
+
+@Entity()
+export class Author {
```

Comment:
Thiện thêm construc vào để có thể build nhanh đối tượng luôn
```suggestion
export class Author {
  constructor(data?: Partial<Author>) {
    data && Object.assign(this, data);
  }
```